configure.ac

```diff
@@ -662,7 +662,7 @@ fi
 # Check for stream support via WinPR
 if test "x${have_freerdp}" = "xyes"
 then
-    AC_CHECK_HEADER(winpr/stream.h,,
+    AC_CHECK_HEADER(winpr/stream.h,[RDP_LIBS="$RDP_LIBS -lwinpr-utils"],
```

Comment:
It would make even *more* sense to add a `WINPR_LIBS` variable, and modify `Makefile.am` accordingly. This is the pattern used elsewhere for libraries which may or may not be present. If they are not present/needed, then the variable just stays empty.